docs/upgrade/index.rst

```diff
@@ -58,15 +56,8 @@ In contrast, a ``NodeTransformer`` (which inherits from a ``NodeVisitor``) is al
 Modifying the AST
 -----------------
 
-
-
-
-
-
-
-
 Technical Backgrounds - Links to External Documentation
----------------------------------------------------------
+.......................................................
```

Comment:
why a level-change of the headings?

Reply:
because it looked like "Technical Backgrounds" is a subsection of "Modifying the AST"